runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -11772,6 +12145,12 @@ J9::X86::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
             return TR::TreeEvaluator::inlineStringLatin1Inflate(node, cg);
             }
          break;
+      case TR::jdk_internal_util_ArraysSupport_vectorizedHashCode:
+         {
+         TR::Register *result = inlineVectorizedHashCode(node, cg);
```

Comment:
You probably have to check if inlining vectorized hashcode is supported before calling.